test/bats/test.bats

```diff
@@ -133,8 +133,8 @@ teardown() {
   wait_for_process $WAIT_TIME $SLEEP_TIME "kubectl get k8srequiredlabels.constraints.gatekeeper.sh ns-must-have-gk -o json | jq '.status.violations[]'"
 
   violations=$(kubectl get k8srequiredlabels.constraints.gatekeeper.sh ns-must-have-gk -o json | jq '.status.violations | length')
-  [[ "$violations" -eq 5 ]]
+  [[ "$violations" -eq 6 ]]
```

Comment:
Why is this updated? kind v0.7.0?

Reply:
that's correct, there is an extra namespace in kind 0.7